File: Android/OTPWA/app/src/main/java/com/example/otpwa/OTP.java
package com.example.otpwa;

import java.util.Date;

public class OTP {
//    private String id;
//    private String number;
//    private String otp;
//    private Date exp;

    private boolean status;

    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public OTP(boolean status) {
        this.status = status;
    }
}

File: Android/otp-wa-main/app/src/main/java/com/example/otpwhatsapp/otp.kt
package com.example.otpwhatsapp

import java.util.*

class otp {
    private lateinit var id: String
    private lateinit var number: String
    private lateinit var otp: String
    private lateinit var exp: Date

    constructor(id: String, number: String, otp: String, exp: Date) {
        this.id = id
        this.number = number
        this.otp = otp
        this.exp = exp
    }

    
}